megamek/src/megamek/common/Compute.java

```diff
@@ -861,9 +861,14 @@ public static Entity findSpotter(IGame game, Entity attacker,
                 los.setTargetCover(LosEffects.COVER_NONE);
                 mods.append(Compute.getAttackerMovementModifier(game,
                                                                 other.getId()));
-                if (other.isAttackingThisTurn() && !other.getCrew().hasActiveCommandConsole()) {
+                
+                // a spotter suffers a penalty if it's also making an attack this round
+                // unless it has a command console or has TAGged the target
+                if (other.isAttackingThisTurn() && !other.getCrew().hasActiveCommandConsole() && 
+                        (!isTargetTagged(null, target, game) || (taggedBy != -1))) {
```

Comment:
This logic isn't quite right, as the target has to be tagged by the specific spotter for the penalty to not apply.